### ISSUE TITLE
Make error

### ISSUE THREAD

**Issue by lamevan** [mrvollger/SDA#7]
Hi,when I install SDA it occured this error...
Can you explain why it occured?Thanks!
  ClobberError: This transaction has incompatible packages due to a shared path.
  packages: defaults::numpy-1.12.1-py27h9378851_1, defaults::numpy-base-1.14.3-py27hdbf6ddf_0
  path: 'lib/python2.7/site-packages/numpy/version.pyc'

**Comment by mrvollger** [comment]
Do you know during which Make rule it broke during?

These are the two most likely candidates:
```
#
# make conda envs
#
envs/python2.done: 
	source env_conda.cfg && \
		mkdir -p envs && \
	   	conda env create --force -f envs/python2.yml --prefix $(PWD)/envs/sda-python-2 && \
	   	touch envs/python2.done

envs/python3.done:
	source env_conda.cfg && \
		mkdir -p envs && \
	   	conda env create --force -f envs/python3.yml --prefix $(PWD)/envs/sda-python-3 && \
	   	touch envs/python3.done
```

**Comment by lamevan** [comment]
Yes..it broke during env/python2.done make rule

**Comment by mrvollger** [comment]
Can you try going into envs/python2.yml and remove`=py27h9378851_1` from the line containing `numpy=.*` and `=py27hdbf6ddf_0` from the line with `numpy-base=.*`.  After this try making again. 

If this does not work then I expect there is a conflict between this and something in your overall python environment.  Did you have other versions of python and or numpy loaded in your env while doing this?

**Comment by lamevan** [comment]
I did what you said and it worked.Thanks!
And when I run SDATEST,it has this error:
Could you explain why this error occured?..
AttributeError: 'list' object has no attribute 'ravel'
[Mon Apr 22 11:56:20 2019]
Error in rule createPSVgraph:
    jobid: 10
    output: CC/mi.gml, CC/mi.adj, CC/mi.mi

RuleException:
CalledProcessError in line 360 of /home/linpeifan/Software/sd/SDA/SDA.1.snakemake.py:
Command ' source /home/linpeifan/Software/sd/SDA//env_python2.cfg;
/home/linpeifan/Software/sd/SDA/CCscripts//PairedSNVs.py snvs/assembly.consensus.fragments.snv.mat.categorized --minCov 27 --maxCov 54          --mi CC/mi.mi --graph CC/mi.gml --adj CC/mi.adj                 --minNShared 5 --minLRT 1.5 --vcf snvs/assembly.consensus.nucfreq.vcf ' returned non-zero exit status 1.
  File "/home/linpeifan/Software/sd/SDA/SDA.1.snakemake.py", line 360, in __rule_createPSVgraph
  File "/home/linpeifan/Software/sd/SDA/envs/sda-python-3/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Removing output files of failed job createPSVgraph since they might be corrupted:
CC/mi.mi
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/linpeifan/Software/sd/SDA/TestCases/SDAtest/.snakemake/log/2019-04-22T115349.859844.snakemake.log

Can't find file /home/linpeifan/.vnc/mu01:96.pid
You'll have to kill the Xvnc process manually

**Comment by lamevan** [comment]
I remove the string you said and it works,thank you!

**Comment by mrvollger** [comment]
Great. If you end up running the test case please let me know if it works. If so I will push this change.

**Comment by lamevan** [comment]
Yes,I have run the test and I found that  wtdbg.assemblies.fasta is an empty file..
If this a mistake?

**Comment by mrvollger** [comment]
For small local assemblies wtdbg2 and miniasm are not as good as canu. So you will often find that the wtdgb2 does not have any output assembly. Personally I always use/trust the canu assembly.

**Comment by mrvollger** [comment]
Fixed with af7126779a712372f788bd4ad4b99cf01f3d3298
